contents/handbook/strategy/strategy.md

```diff
@@ -4,40 +4,43 @@ sidebar: Handbook
 showTitle: true
 ---
 
-> PostHog enables product led growth.
+<br>
+
+> PostHog follows the philosophy of product-led growth.
+<br>
 
 ## Where will we compete?
 
-PostHog wish to provide best in class product analytics, by appealing to developer and product teams.
+PostHog aims to provide best-in-class product analytics, by appealing to Engineering and Product teams.
 
-We will target developers, rather than product teams to get implemented. There are a few key aspects to this:
+We will target developers rather than product teams in order to get implemented. There are a few key aspects to this:
 
-* An open course business model
- * We have a zero friction route to adoption in both small, mid size and big companies.
- * We can go for breadth over depth in the product, similar to GitLab, letting the community fill in the gaps.
- * Being open core means we can aim for mass adoption with our free version, without worrying about competition from hosting companies.
+* An open core business model
+  * We have a zero friction route to adoption in both small, medium size, and big companies.
+  * We can go for breadth over depth in the product, similar to GitLab, letting the community fill in the gaps.
+  * Being open core means we can aim for mass adoption with our free version, without worrying about competition from companies offering hosting services.
 * Tighter integration with developers' environments.
- * Integration with git repo (feature flagging) & integration of data into local environment (terminal/localhost)
- * Working with existing databases and pipelines.
- * 1 click / 1 line install into production.
-* Defensive product-team strategy - stay at parity with their features, but don't innovate here.
- * This means execution is much faster.
- * It will prevent most churn due to missing features from major tools that product teams will push for.
+  * Integration with git repo (feature flagging) & integration of data into local environment (terminal/localhost)
+  * Working with existing databases and pipelines.
+  * 1-click / 1-line install into production.
+* Defensive Product team strategy - stay at parity with their features, but don't innovate here.
+  * This means execution is much faster.
+  * It will prevent most churn due to missing features from major tools that product teams will push for.
 
 ## How will we be the best?
 
-* Engineering-led culture. Engineers want to deal with engineers.
-* We will make the company remote-first, to encourage the greatest possible degree of community collaboration.
+* **Engineering-led culture:** Engineers want to deal with engineers.
+* **Remote first:** We will make the company remote-first, to encourage the greatest possible degree of community collaboration.
```

Comment:
All-remote I think is a more accurate term. Remote-first usually indicates some office somewhere, which we will never have